docs/CONTRIBUTING.md

```diff
@@ -94,6 +92,116 @@ Now force push to update your PR:
 git push --force
 ```
 
+## API tests
+
+There are two kinds of tests in the API: unit ([`phpunit`](https://phpunit.de)) and integration
+([`behat`](http://behat.org)) tests.
+
+Both `phpunit` and `behat` are development dependencies and should be available in the `vendor` directory.
+
+### PHPUnit and coverage generation
+
+To launch unit tests:
+
+```bash
+docker-compose exec php bin/phpunit
+```
+
+If you want coverage, you will need the `phpdbg` package and run:
+
+```bash
+docker-compose exec php phpdbg -qrr bin/phpunit --coverage-html coverage
+```
+
+Coverage will be available in `api/coverage/index.html`.
+
+### Behat
+
+To launch Behat tests:
+
+```bash
+docker-compose exec php bin/behat
+```
+
+You may need to clear the cache manually before running Behat tests because of the temporary database. To do so, just
+remove the `test` cache directory:
+
+```bash
+docker-compose exec php rm -r var/cache/test
+```
+
+### Doctrine schema validation
+
+To analyse your Doctrine schema, use:
+
+```bash
+docker-compose exec php bin/console doctrine:schema:validate --skip-sync
+```
+
+### Security checker
+
+To check security issues in project dependencies, use:
+
+```bash
+docker-compose exec php bin/security-checker security:check
+```
+
```

Comment:
`See details at the [documentation](https://github.com/sensiolabs/security-checker).`